warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/IngestJob.java

```diff
@@ -247,6 +247,7 @@ protected void printUsage() {
     
     @Override
     public int run(String[] args) throws Exception {
+        inputFormat = EventSequenceFileInputFormat.class;
```

Comment:
This is already initialized to the EventSequenceFileInputFormat in the construction of the class.  Is this needed?

Reply:
This was just a placeholder.